receiver/sqlserverreceiver/README.md

```diff
@@ -52,6 +52,8 @@ sqlserver:
   username: myusername
   password: mypassword
   server: sqlserver.address
+  datasource: sqlserver://yourserver.database.windows.net?database=yourdb&fedauth=ActiveDirectoryMSI
+  use_azure_ad: true
```

Comment:
With the changes from this PR the config example is invalid (`datasource` can't be used with `username`, etc.).

If we need to include an example making use of `datasource` and `use_azure_ad`, I'd suggest adding an example under `./testdata` and simply referencing it in the README.